changelog/unreleased/radio-acessibility

```diff
@@ -0,0 +1,5 @@
+Enhancement: Add aria properties to radio input
```

Comment:
Please rename the changelog file to `enhancement-radio-accessibility` (add `enhancement-` prefix and fix typo in `accessibility`)